ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,24 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New Fields for PAC (ACU) Units
+- return_air_isolation_damper_status
+	- OPEN
+	- CLOSE
+- alert_reset_command
+	- NORMAL
+	- RESET
+- fan_run_time_accumulator
+- compressor_run_time_accumulator
+- evaporative_fan_speed_sensor
```

Comment:
is this an evaporative fan or an evaporator fan?

Reply:
Its is an evaporator fan (Evaporating fan). The sub-field has "evaporative" describing s a process of evaporation of water.